openapi-cli/src/test/resources/generators/client/ballerina/multipart_formdata.bal

```diff
@@ -47,4 +47,18 @@ public isolated client class Client {
         http:Response response = check self.clientEp->post(resourcePath, request);
         return response;
     }
+
+    # Create an user
+    #
+    # + payload - User
+    # + return - Null response
+    remote isolated function createUser(User_body payload) returns http:Response|error {
+        string resourcePath = string `/user`;
+        http:Request request = new;
+        map<Encoding> encodingMap = {};
```

Comment:
I doubt that whether we need to add this line 58 for code generation since the given sample encode file didn't mention which fields should be encoded. 
@aneeshafedo WDYT?

Reply:
Fixed with [Avoid encoding map creating if the encoding information is absent](https://github.com/ballerina-platform/openapi-tools/pull/1144/commits/14a86ae2b79bd9c01ab34be863ca2b87781db6e1)